src/main/java/games/strategy/engine/lobby/server/db/DbUserController.java

```diff
@@ -62,8 +75,9 @@ public boolean doesUserExist(final String userName) {
   @Override
   public void updateUser(final DBUser user, final HashedPassword password) {
     Preconditions.checkArgument(user.isValid(), user.getValidationErrorMessage());
-    Preconditions.checkArgument(password.isValidSyntax());
-    primaryDao.updateUser(user, password);
+    if (!password.isBcrypted()) {
```

Comment:
Cool, okay, fair enough here having `HashedPassword` carry a flag that dictates control flow.  :+1: